src/rt/trace.d

```diff
@@ -141,13 +141,15 @@ static Stack* stack_malloc()
     }
     else
         s = cast(Stack *)trace_malloc(Stack.sizeof);
```

Comment:
so this code has no {} but the previous one has (compounded with the horrific `{` followed by the stuff). So use one or the other but not all three in one.